payments/vouchers.go

```diff
@@ -0,0 +1,67 @@
+package payments
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/crypto"
+	nitroAbi "github.com/statechannels/go-nitro/abi"
+	"github.com/statechannels/go-nitro/channel/state"
+	nitroCrypto "github.com/statechannels/go-nitro/crypto"
+	"github.com/statechannels/go-nitro/types"
+)
+
+type (
+	// A Voucher signed by Alice can be used by Bob to redeem payments in case of
+	// a misbehaving Alice.
+	//
+	// During normal operation, Alice & Bob would terminate the channel with an
+	// outcome reflecting the largest amount signed by Alice. For instance,
+	// - if the channel started with balances {alice: 100, bob: 0}
+	// - and the biggest voucher signed by alice had amount = 20
```

Comment:
So vouchers are "replace by incentive", right? As in, Bob can only redeem one voucher per channel?

Reply:
That's correct